cmd/main.go

```diff
@@ -76,7 +77,7 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.BoolVar(&certManagerEnabled, "cert-manager-enabled", false, "Enable cert-manager integration")
-	flag.BoolVar(&webhookEnabled, "webhook-enabled", true, "Enable CRDs conversion webhook.")
+	flag.BoolVar(&webhookEnabled, "webhook-enabled", false, "Enable CRDs conversion webhook.") //TODO Revert
```

Comment:
```suggestion
	flag.BoolVar(&webhookEnabled, "webhook-enabled", true, "Enable CRDs conversion webhook.")
```

If you use vscode, not sure for the other, you can set argument when running the operator in debug mode.